### ISSUE TITLE
Please add server to list

### ISSUE THREAD

**Issue by pallebone** [aeharding/voyager#1452]
Hello,

i read on your front page you can request your server to be added to to voyager. 

If this is indeed possible, please can I ask my server be added?

https://lemmy.myserv.one/

Thank you either way. 

Kind regards
Peter

**Comment by aeharding** [comment]
Hi there!

I can add your server to the login autocomplete list if you'd like!

For the signup page, there are a few baseline requirements

https://github.com/aeharding/voyager/blob/b11492574e4cf94fc57a33aefdbf0017884da21e/src/features/auth/login/data/README.md#L4

The main obstacle is the 50 MAU (you're halfway there)

So LMK if you want to be added to the login list, and then down the road once you address the signup criteria I can add you to that list as well!

**Comment by pallebone** [comment]
Ok that is totally fine, thank you.

**Comment by pallebone** [comment]
Thanks :)

**Comment by pallebone** [comment]
@aeharding I have been going at this for months and I am still only at 30 users per month. Do we really have to get to 50 users? This is really hard without exposure in an app like other instances and I feel like I just am falling behind.

**Comment by pallebone** [comment]
@aeharding Im now at 42 😢

**Comment by aeharding** [comment]
Hi @pallebone,

Can you complete the other requirements? Then I will add your instance

- Commitment to provide 2 months lead time before shut down
- Active moderation against harassment, bullying, racism, discrimination, transphobia, hate speech, violation of privacy and threats of violence.
- Privacy Policy and Terms of Use set up on /legal page

**Comment by pallebone** [comment]
Yes first 2 issues is no problem whatsoever. I am already also on the matrix chat for lemmy world Defence HQ and follow their rules so this is no problem.

Regards the legal page, I will have to make one and come back to you.

Thank you.

Pete

**Comment by pallebone** [comment]
Thank you :)
